### DISCUSSION TITLE
Is there a possibility to get this working with oobabooga/text-generation-webui ?

### DISCUSSION THREAD

**Discussion by ksylvan** [PromtEngineer/localGPT#26] [Ideas]
It would be great if we could merge some of this code into the text-generation-webui  so that you could then...

1. drop your files/documents into a folder, then
2. bring up the webui to have a conversation about your documents.

https://github.com/oobabooga/text-generation-webui

**Comment by PromtEngineer** [comment]
I haven't looked at the creating extensions for Oobabooga text-generation-webui but I agree that would be great. Will look into it or someone from the community can help with that :)
